lib/src/accordion/Accordion.tsx

```diff
@@ -83,162 +110,112 @@ const DXCAccordion = styled.div`
-      }
-    }
-    .MuiButtonBase-root {
-      border-radius: ${(props) => props.theme.borderRadius};
-
-      &.Mui-expanded {
-        border-bottom-right-radius: 0;
-        border-bottom-left-radius: 0;
-        .MuiSvgIcon-root {
-          opacity: 1;
-        }
-      }
-      &.MuiIconButton-root {
-        height: auto;
-      }
-      .MuiExpansionPanelSummary-content {
-        padding-top: ${(props) => props.theme.titleLabelPaddingTop};
-        padding-bottom: ${(props) => props.theme.titleLabelPaddingBottom};
-        padding-right: ${(props) => props.theme.titleLabelPaddingRight};
-        padding-left: ${(props) => props.theme.titleLabelPaddingLeft};
-        min-width: 0;
-        &.Mui-expanded {
-          div:nth-child(2) {
-            opacity: 1;
-          }
-        }
-        :hover {
-          div {
-            opacity: 1;
-          }
-        }
-      }
-    }
-    .MuiTouchRipple-root {
-      display: none;
-    }
-  }
-  .MuiCollapse-hidden {
-    display: none;
-  }
-  .MuiCollapse-container {
-    border-radius: 0px 0px 4px 4px;
-    cursor: default;
-    width: 100%;
+`;
+
+const AccordionHeader = styled.h3`
+  display: flex;
+  flex-direction: column;
+  min-height: 48px;
+  margin: 0;
+`;
+
+const AccordionTrigger = styled.button<{ isExpanded: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 24px;
+  width: 100%;
+  background-color: transparent;
+  border: none;
+  border-radius: ${(props) => props.theme.borderRadius};
+  ${(props) => props.isExpanded && `border-bottom-left-radius: 0; border-bottom-right-radius: 0;`}
+  padding: 12px 16px;
+  cursor: ${(props) => (props.disabled ? "not-allowed" : "pointer")};
+
+  :focus {
+    outline: ${(props) =>
+      `${props.theme.focusBorderColor} ${props.theme.focusBorderStyle} ${props.theme.focusBorderThickness}`};
   }
-  .MuiIconButton-label {
-    & > .MuiSvgIcon-root {
-      color: ${(props) => (props.disabled ? props.theme.disabledArrowColor : props.theme.arrowColor)};
-    }
+  :hover:enabled {
+    background-color: ${(props) => `${props.theme.hoverBackgroundColor}`};
   }
-  .MuiExpansionPanelDetails-root {
-    padding: ${(props) => (props.padding && typeof props.padding !== "object" ? spaces[props.padding] : "0px")};
-    padding-top: ${(props) =>
-      props.padding && typeof props.padding === "object" && props.padding.top ? spaces[props.padding.top] : ""};
-    padding-right: ${(props) =>
-      props.padding && typeof props.padding === "object" && props.padding.right ? spaces[props.padding.right] : ""};
-    padding-bottom: ${(props) =>
-      props.padding && typeof props.padding === "object" && props.padding.bottom ? spaces[props.padding.bottom] : ""};
-    padding-left: ${(props) =>
-      props.padding && typeof props.padding === "object" && props.padding.left ? spaces[props.padding.left] : ""};
+  :active:enabled {
+    background-color: ${(props) => `${props.theme.hoverBackgroundColor}`};
```

Comment:
As in the guidelines, active state should also have outline.

Reply:
This wouldn't make any difference apart from a storie level. The` :active` selector works from the moment the component has been clicked to the mouse release, which coincides with the focus-gaining moment and, therefore, the `:focus` is activated.

Reply:
You can try this by long pressing the component. The design guidelines where trying the mouse-pressed state, which has the focus, as our component.